modules/serverless-installing-cli-web-console.adoc

```diff
@@ -6,12 +6,12 @@
 [id="installing-cli-web-console_{context}"]
 = Installing the Knative CLI using the {product-title} web console
 
-You can use the following procedure to install the `kn` CLI by using the web console.
+Using the {product-title} web console provides a streamlined and intuitive user interface to install the Knative (`kn`) CLI. After the {ServerlessOperatorName} is installed, you will see a link to download the Knative (`kn`) CLI for Linux (x86_64, amd64, s390x, ppc64le), macOS, or Windows from the *Command Line Tools* page in the {product-title} web console.
```

Comment:
I know we're copying this bit from the old one, but anyway, we have only 3 Linux architectures, not 4, as `x86_64` and `amd64` are synonyms, so we should perhaps only say `(amd64, s390x, ppc64le)` (as the binary for x86_64 is called `amd64` in https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest/ )